src/Symfony/Component/VarDumper/Caster/ResourceCaster.php

```diff
@@ -29,7 +29,7 @@ class ResourceCaster
      */
     public static function castCurl(\CurlHandle $h, array $a, Stub $stub, bool $isNested): array
     {
-        trigger_deprecation('symfony/var-dumper', '7.3', 'The "%s()" method is deprecated without replacement.', __METHOD__, CurlCaster::class);
+        trigger_deprecation('symfony/var-dumper', '7.3', 'The "%s()" method is deprecated without replacement.', __METHOD__);
```

Comment:
I think we should not even use the extra arguments there but use an explicit `\sprintf` call instead, which can be optimized by OPCache at compile time on recent PHP versions as we only use `%s` placeholders (the optimization supports `%s` and `%d`).

Reply:
And as another benefit, static analysis tools are able to detect such issues about mismatch for the number of parameters.